apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts

```diff
@@ -0,0 +1,54 @@
+/* istanbul ignore file */
+import { RestTeam, RestUser } from '@asap-hub/squidex';
+import { Migration } from '../handlers/webhooks/webhook-run-migrations';
+import { applyToAllItemsInCollection } from '../utils/migrations';
+
+export default class MoveRepurposedFields extends Migration {
+  up = async (): Promise<void> => {
+    await migrateUserFields();
+    await migrateTeamFields();
```

Comment:
maybe break it into two separate migrations?

Reply:
agreed to leave as is